File: app/src/main/java/com/shevelev/comics_viewer/dal/dto/Option.java
package com.shevelev.comics_viewer.dal.dto;

import com.shevelev.comics_viewer.dal.entities.DbOption;

/**
 * One innerOptionsCollection record
 */
public class Option
{
    public int key;

    public String value;

    public Option(int key, String value)
    {
        this.key = key;
        this.value = value;
    }

    public Option(DbOption option)
    {
        this.key = option.key;
        this.value = option.value;
    }
}

File: app/src/main/java/com/shevelev/comics_viewer/dal/dto/Option.kt
package com.shevelev.comics_viewer.dal.dto

import com.shevelev.comics_viewer.dal.entities.DbOption

/**
 * One innerOptionsCollection record
 */
class Option {
    var key: Int
    var value: String?

    constructor(key: Int, value: String) {
        this.key = key
        this.value = value
    }

    constructor(option: DbOption) {
        key = option.key
        value = option.value
    }
}